requirements/3.11/requirements.txt

```diff
@@ -0,0 +1,4 @@
+distro==1.9.0
+PyYAML==6.0.2
+ruamel.yaml==0.18.7
+six==1.17.0
```

Comment:
Why do we have different versions of six between Py 3.7 and 3.11?
Was is the case already before your change?
If not, I would align them all to the same version since we will get rid of that soon.